services/horizon/internal/actions/helpers.go

```
@@ -45,6 +46,13 @@ func (base *Base) GetCursor(name string) string {
 		cursor = lei
 	}
 
+	// In case cursor is negative value, return InvalidField error
+	cursorInt, _ := strconv.Atoi(cursor)
```

It's always weird to ignore an error outside of example go, so instead let's do this:

```go
cursorInt, err := ...
if err == nil && cursorInt < 0 {
   ...
}
```